ribs.py

```diff
@@ -105,12 +105,30 @@ def key_pressed(key):
            (keycode not in last_frame_held_buttons)
 
 
+#
+# Simple sprite drawing
+#
+
+def draw_centered(img, x, y):
```

Comment:
Should we add a scale and rotation parameter to this? Feels like something that would come in handy

Reply:
I wondered if I should add it or not. I decided against it to have simpler functions, but people seem to love their rotations and scales...

Reply:
Seems like something people would want to do for bullets etc. We could keep both though

Reply:
Maybe adding them as optional parameters so they don't have to think about them if they don't explicitly want to use them

Reply:
Oh right, those are a thing in python. Sounds like a good idea!

Reply:
The thing is, optional paramters are a bit confusing... Maybe not tell them about edge cases?

Also, I already added this when you wrote this comment. :)